data-pipeline/src/pipeline/pre_processing/ancillary/local_authority.py

```diff
@@ -25,8 +29,20 @@ def build_local_authorities(
         year,
     )
 
-    # TODO: combined with out LA datasets.
-    return section_251_data
+    logger.info("Processing Local Authority statistical neighbours data.")
+
+    la_statistical_neighbours_data = _prepare_la_statistical_neighbours(
+        statistical_neighbours_filepath, year
+    )
+
+    local_authority_data = section_251_data.merge(
+        la_statistical_neighbours_data,
+        left_on="old_la_code",
+        right_index=True,
+        how="left",
```

Comment:
question: does the `section_251_data` need a `reset_index()` before doing this? IIRC, it's got a multi-index of `old_la_code` and `new_la_code` which won't match…?

Reply:
No, all good: if Pandas is behaving itself (and I'll admit _that's_ the surprising part) it's good as it is.